dom0/sd-rsyslog.conf.j2

```diff
@@ -0,0 +1,3 @@
+[sd-rsyslog]
```

Comment:
:+1: templating

Reply:
Templating definitely good. We might be able to use `grains["localhost"]` to get the VM name, rather than passing in `context` every time. (See output of e.g. `sudo qubesctl --skip-dom0 --show-output --target sys-firewall grains.items`.) If that works, we could stick the tasks for 1) install pkg; 2) configure rsyslog in a single state file and include it, reducing a lot of duplication.